### PR TITLE
ENG-10829: Deprecate using catalog for YCSB.

### PR DESCRIPTION
**Pull request by yt14**
Changes made:

- Remove schema="catalog" in deployment.xml 
- Update run.sh to compile and load jars and DDL. 
- Load jars in ycsb_ddl.sql and batch load procedures. 
- Create README.md and rewrite readme, to reflect changes and prepare to add our driver to YCSB repo.

### CODE REVIEW

File: tests/test_apps/ycsb/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		./run.sh server [leader]
+
+For example, to start a cluster with 3 nodes (host0(leader), host1, host2), type following command on those hosts:
+
+		./run.sh server host0
+
+If leader host is not provided, by default "localhost" is used.
+
+You can specify other server configurations such as hostcount, sitesperhost or kfactor in deployment.xml.
+
```
Comment (nshi) [inline tests/test_apps/ycsb/README.md]: Do you want to mention `./run.sh init` here?

File: tests/test_apps/ycsb/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+		export YCSB_HOME="<directory where you extracted the above tarball>"
+
+Start VoltDB Server
+--------------------
+To start or add a server to your cluster, invoke run.sh with the "server" parameter, passing the name of the leader host as the second parameter:
+
+		./run.sh server [leader]
+
+For example, to start a cluster with 3 nodes (host0(leader), host1, host2), type following command on those hosts:
```
Comment (nshi) [inline tests/test_apps/ycsb/README.md]: To do this, user has to change the deployment.xml file, too, right?

**Comment by nshi** [comment]
Review finished.

**Comment by yt14** [comment]
Comments addressed.

**Pull request merged**